configure.ac

```diff
@@ -482,6 +482,9 @@ fi
 # For hddtemp module
 AC_CHECK_HEADERS(linux/major.h)
 
+# For hugepages module
+AC_CHECK_HEADERS(dirent.h)
```

Comment:
You're not doing anything with the result of this check.
Best to disable the hugepages plugin if this header is not found.